backends/verilog/verilog_backend.cc

```diff
@@ -2345,6 +2353,7 @@ struct VerilogBackend : public Backend {
 				continue;
 			}
 			log("Dumping module `%s'.\n", module->name.c_str());
+			if (toposort) module->run_toposort_cells();
```

Comment:
In my opinion, a backend should have no side effects on the design.

Reply:
Agreed: sometimes it can be useful to output to multiple formats, and there shouldn't be order-dependency - we have enough bug reports about chaos in Yosys as it is.